spec/mailers/supervisor_mailer_spec.rb

```diff
@@ -116,4 +116,16 @@
       expect(email_body).to include("This invitation will expire on #{expiration_date} (two weeks).")
     end
   end
+
+  describe ".reimbursement_request_email" do
+    let(:supervisor) { create(:supervisor, receive_reimbursement_email: true) }
+    let(:volunteer) { create(:volunteer) }
+    let(:casa_organization) { volunteer.casa_org }
+
+    let(:mail) { SupervisorMailer.reimbursement_request_email(volunteer, supervisor) }
+
+    it "sends email reminder" do
+      expect(mail.body.encoded).to match("#{volunteer.display_name} has submitted a reimbursement request")
```

Comment:
Could you also check that the `to:` and the `subject:` are correct?

Reply:
Have done the changes to check to and subject as well.